### ISSUE TITLE
AncientCoderPack and Linux

### ISSUE THREAD

**Issue by d34dinsid3k1tty** [ancientmc/AncientCoderPack#2]
Hiya!

At first thank you for creating it, I plan use your's toolchain for creating Alpha 1.2.6 mods, I never was to able force original Minecraft Coder Pack for a1.2.6 to work on Linux due to elder codebase.

However, I tried to `gradle setup` to build the whole ACP, but sadly I got `cfg\mappings\a1.2.6.tsrg (No such file or directory)` despites it's exists.

![image](https://github.com/user-attachments/assets/3f3f3b96-b7fa-4c91-8e13-bb2bdcd86b00)

After a moment of thinkering I reminded that `cfg\mappings\a1.2.6.tsrg` don't exists because it's not either valid Linux paths because Linux use slashes, not backslashes.

If it possible to create ACP support for Linux? Or I can use some workarounds for it? I really need this Coder Pack to create a1.2.6 mod.

Thank you for your amazing work and for responding to me. 🌸

Much loves. 💖

**Comment by moist-mason** [comment]
Hi there! Thanks for checking out my project. Regarding your issue, I made a commit https://github.com/ancientmc/AncientCoderPack/commit/e3b68a1a9f7e50744cf44086ff5d1ea434b52414 and also commented on your pull request. Gradle on Windows lets you use '/' as a path separator so as long as that's used instead of '\\' it should work.

**Comment by d34dinsid3k1tty** [comment]
Great! Then I can expect fixes in rest of tool? (acp-gradle for e.g.) I really need this on my Linux workstation. Thank you

**Comment by moist-mason** [comment]
> Great! Then I can expect fixes in rest of tool? (acp-gradle for e.g.) I really need this on my Linux workstation. Thank you

Yes. It should just be a matter of replacing any instance of the "\\\\" separator with "/" in the source, since that would allow both Windows and Unix compatability.
